test/acceptance/test/install_tests.go

```diff
@@ -131,6 +153,78 @@ var _ = Describe("Weave GitOps Install Tests", func() {
 		})
 	})
 
+	It("Verify that gitops can install & uninstall gitops components to multiple clusters under a user-specified namespace via pull requests", func() {
+		namespace := "test-namespace"
+
+		_, cluster1Context, err := ResetOrCreateCluster(namespace, true)
```

Comment:
nit: test is a bit easier to read if the context is named clusterPRContext and clusterAutoMergeContext (or something to indicate the actual difference between the contexts)

Reply:
Actually is using PR to install both clusters and auto-merge for uninstall